koku/reporting/provider/gcp/sql/views/reporting_gcp_cost_summary.sql

```diff
@@ -0,0 +1,22 @@
+DROP INDEX IF EXISTS gcp_cost_summary;
+DROP MATERIALIZED VIEW IF EXISTS reporting_gcp_cost_summary;
+
+CREATE MATERIALIZED VIEW reporting_gcp_cost_summary AS(
+    SELECT row_number() OVER(ORDER BY usage_start, source_uuid) as id,
+        usage_start,
+        usage_start as usage_end,
+        sum(unblended_cost) as unblended_cost,
+        sum(markup_cost) as markup_cost,
+        currency
+        source_uuid
+    FROM reporting_gcpcostentrylineitem_daily_summary
+    -- Get data for this month or last month
+    WHERE usage_start >= DATE_TRUNC('month', NOW() - '1 month'::interval)::date
+    GROUP BY usage_start, source_uuid, currency
```

Comment:
```suggestion
    GROUP BY usage_start, source_uuid
```

Reply:
You can duplicate this change across the board so you don't have to group by the currency term.